api/api.go

```diff
@@ -292,3 +292,28 @@ func (c *Client) Skip(language, slug string) error {
 
 	return errors.New(pe.Error)
 }
+
+// Status sends a request to exercism to fetch the user's
+// completion status for the given language track.
+func (c *Client) Status(language string) (*StatusInfo, error) {
```

Comment:
@kytrinyx I don't understand what you mean with your comment above. Can you elaborate a bit, please?